src/braket/device_schema/dwave/dwave_2000Q_device_level_parameters_v1.py

```diff
@@ -84,3 +84,14 @@ class Dwave2000QDeviceLevelParameters(BraketSchemaBase):
     reinitializeState: Optional[bool]
     resultFormat: Optional[ResultFormat]
     spinReversalTransformCount: Optional[int] = Field(gt=0)
+
+    class Config:
+        @staticmethod
+        def schema_extra(schema, model):
+            for prop, value in schema.get("properties", {}).items():
+                if prop != "braketSchemaHeader":
+                    try:
+                        was = value["type"]
+                        value["type"] = [was, "null"]
+                    except KeyError:
+                        pass
```

Comment:
Consider "DRYing" (Don't Repeat Yourself) the code here, but this makes sense for readability.